### ISSUE TITLE
NextJs incompatibility: Global CSS cannot be imported from within node_modules.

### ISSUE THREAD

**Issue by JohnMealy23** [JakeHartnell/react-images-upload#192]
Looks like NextJs is incompat with your module's method for including CSS.  I get the following error when attempting to include your package.

```
Failed to compile
./node_modules/react-images-upload/index.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/react-images-upload/compiled.js
```

More of an FYI, though.  Thanks for contributing to open source, and feel free to close this out if you have no interest in fixing!

**Comment by goffxnca** [comment]
Same problem

**Comment by sodatat** [comment]
Importing CSS files within node_modules cannot be supported
[https://github.com/vercel/next.js/blob/master/errors/css-npm.md](https://github.com/vercel/next.js/blob/master/errors/css-npm.md)

**Comment by sebastiancrossa** [comment]
Running into the same issue

**Comment by JakeStanger** [comment]
Yep, same here. The best solution imo is to simply remove the import from `index.js` and then require users to import it themselves. This would also enable users to ship their own styles.

**Comment by jaywcjlove** [comment]
https://github.com/uiwjs/react-md-editor/issues/52#issuecomment-848969341

Perfect solution. @sodatat @JohnMealy23
